### PR TITLE
feat: UNT-T10302 Implement Redux Toolkit on home screen

### PR DESCRIPTION
**Pull request by vishva-simform**
- Implement UI for HomeScreen

### CODE REVIEW

File: app/components/DropDown/DropDownStyles.ts [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+  moderateScale,
+  verticalScale,
+} from '../../theme';
+
+export const styles = StyleSheet.create({
+  dropDownContainer: {
+    position: 'absolute',
+    right: 0,
+    backgroundColor: Colors.lightGreen,
+    borderRadius: 20,
```
Comment (Krunal-K-SimformSolutions) [inline app/components/DropDown/DropDownStyles.ts:14]: used theme scale functions

File: app/components/DropDown/DropDownStyles.ts [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+  },
+  dropDownButton: {
+    flexDirection: 'row',
+    justifyContent: 'space-between',
+    alignItems: 'center',
+    backgroundColor: Colors.DarkBlue,
+    width: horizontalScale(145),
+    paddingVertical: verticalScale(4),
+    paddingHorizontal: horizontalScale(15),
+    borderRadius: 15,
```
Comment (Krunal-K-SimformSolutions) [inline app/components/DropDown/DropDownStyles.ts:25]: used theme scale functions

File: app/components/DropDown/DropDownStyles.ts [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+  },
+  dropDownStyle: {
+    flexDirection: 'row',
+    justifyContent: 'space-between',
+    alignItems: 'center',
+  },
+  chevronIconStyle: {
+    tintColor: Colors.lightGreen,
+    height: 15,
+    width: 15,
```
Comment (Krunal-K-SimformSolutions) [inline app/components/DropDown/DropDownStyles.ts]: used theme scale functions

File: app/constants/staticData.ts [hunk 4]
```diff
@@ -1,9 +1,10 @@
 import { Icons } from '../assets';
 import { Community, HomeScreen, Movies, Trailers } from '../modules';
+import type { ItemType } from '../navigation/Types';
```
Comment (Krunal-K-SimformSolutions) [inline app/constants/staticData.ts]: import from navigation index file

File: app/services/Utils.ts [hunk 5]
```diff
@@ -0,0 +1,15 @@
+import type { ResultType } from '../components/List';
```
Comment (Krunal-K-SimformSolutions) [inline app/services/Utils.ts]: import from components index file

**Comment by Krunal-K-SimformSolutions** [comment]
missing assets for largeLogo.png

File: app/components/DropDown/DropDown.tsx [hunk 6]
```diff
@@ -0,0 +1,60 @@
...
+          style={[styles.buttonStyle, dropDownButtonStyle]}>
+          {selectedItem.name}
+        </Text>
+        <Image
+          source={Icons.chevron}
+          style={[styles.chevronIconStyle, latestTrailerChevron]}
+        />
+      </TouchableOpacity>
+      {showOptions && (
+        <View>
```
Comment (Krunal-K-SimformSolutions) [inline app/components/DropDown/DropDown.tsx]: Used fragment(<>) when not used any props of view.

File: app/components/DropDown/DropDownStyles.ts (hunk 1 above)
Comment (vishva-simform) [reply app/components/DropDown/DropDownStyles.ts:14]: We are instructed that when using border radius don't use any scale functions , and also when right/left is defined to 0 no need to use scale functions

File: app/components/DropDown/DropDownStyles.ts (hunk 2 above)
Comment (vishva-simform) [reply app/components/DropDown/DropDownStyles.ts:25]: We are instructed that when using border radius don't use any scale functions

**Review by Krunal-K-SimformSolutions — APPROVED** [review]

**Pull request merged**